### PR TITLE
Update repo Issue templates

### LINKED ISSUE #4520: Add Issue Templates
The [new Issue Page](https://github.com/dotnet/aspire/issues/new/choose) currently makes it look like you're only interested in Api proposals and security vulnerabilities.  Yes there is a "Open a blank issue" option but it's a bit hidden
image

Is it worth adding at least Feature & Bug templates so the New Issue page is a bit more welcoming, e.g. [aspnetcore](https://github.com/dotnet/aspnetcore/issues/new/choose)
image

### PR DESCRIPTION
**Pull request by timheuer**
Fix #4520 copying similar issue/feature templates from aspnetcore and adding config.yml
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/4633)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      label: .NET Version info
+      description: |
+        Run `dotnet --info`
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Anything else?
+      description: |
+        - ASP.NET Core version:
```
Comment (danmoseley) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:63]: Aspire version?

Comment (timheuer) [reply .github/ISSUE_TEMPLATE/10_bug_report.yml:63]: Probably both good prompts, added.

**Review by maddymontaquila — APPROVED** [review]
lgtm!

**Review by danmoseley — APPROVED** [review]
can you do a quick test using your fork? looks like you're not using a fork, so you'd put this branch in your fork I think that's something like `git push yourfork dev/timheuer/4520` where yourfork is the remote to it. then change default branch in repo settings of fork to dev/timheuer/4520. then click new issue in the repo.

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+
+        Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+
+        If you believe you have an issue that affects the SECURITY of the platform, please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) but ONLY if it is reported through email.
+        For other types of questions, consider using [Discussions](https://github.com/dotnet/aspnetcore/discussions).
```
Comment (danmoseley) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:12]:
```suggestion
        For other types of questions, consider using [Discussions](https://github.com/dotnet/aspire/discussions).
```

**Comment by timheuer** [comment]
Found one typo (fixed) in a URL for discussions.  Else all looked good:

**New Issue Dialog**
image

**Bug**
image

**Feature Request**
image

**Links**
API Proposal remains unchanged
Other links worked (Discussions, Security, Issue Policies)

File: .github/ISSUE_TEMPLATE/20_feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+description: Suggest an idea for this project
+body:
+  - type: markdown
+    attributes:
+      value: |
+        More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the feature you are requesting. (https://github.com/dotnet/aspnetcore/issues).
```
Comment (danmoseley) [inline .github/ISSUE_TEMPLATE/20_feature-request.yml:11]:
```suggestion
      description: Please search to see if an issue already exists for the feature you are requesting. (https://github.com/dotnet/aspire/issues).
```
I really should not have reviewed on my phone ..

Comment (timheuer) [reply .github/ISSUE_TEMPLATE/20_feature-request.yml:11]: Found already and fixed ;-)

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+
+        Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+
+        If you believe you have an issue that affects the SECURITY of the platform, please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) but ONLY if it is reported through email.
+        For other types of questions, consider using [Discussions](https://github.com/dotnet/aspire/discussions).
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered ([aspnetcore/issues](https://github.com/dotnet/aspire/issues?q=is%3Aissue)). More information on our issue management policies is available [here](https://aka.ms/aspnet/issue-policies).
```
Comment (radical) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:16]:
```suggestion
      description: Please search to see if an issue already exists for the bug you encountered ([aspire/issues](https://github.com/dotnet/aspire/issues)). More information on our issue management policies is available [here](https://aka.ms/aspnet/issue-policies).
```

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: |
+        We ❤ code! Point us to a [minimalistic project which reproduces this issue (repro)](https://github.com/dotnet/aspnetcore/blob/main/docs/repro.md) hosted in a public GitHub repo.
```
Comment (radical) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:36]: `https://github.com/dotnet/aspnetcore/blob/main/docs/repro.md`
Do we need a copy/variation of this doc in `aspire`?

File: .github/ISSUE_TEMPLATE/10_bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      label: Exceptions (if any)
+      description: Include the exception you get when facing this issue.
+      placeholder:
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: .NET Version info
+      description: |
+        Run `dotnet --info`
```
Comment (radical) [inline .github/ISSUE_TEMPLATE/10_bug_report.yml:56]: Would getting the `aspire` workload version also be useful?

**Comment by danmoseley** [comment]
LGTM!

**Review by joperezr — APPROVED** [review]
This looks great to me, thanks @timheuer !

**Review by JamesNK — APPROVED** [review]
:shipit:

File: .github/ISSUE_TEMPLATE/10_bug_report.yml (hunk 6 above)
Comment (timheuer) [reply .github/ISSUE_TEMPLATE/10_bug_report.yml:56]: That comes in with the dotnet --info call

**Review by radical — APPROVED** [review]

**Comment by radical** [comment]
Thank you for adding this, @timheuer!

**Comment by danmoseley** [comment]
Some good feedback, but my 2c -- I suggest to put this in and iterate based on feedback. We don't get a huge number of issues, there's still a blank template, it's easy to change...

**Comment by radical** [comment]
> Some good feedback, but my 2c -- I suggest to put this in and iterate based on feedback. We don't get a huge number of issues, there's still a blank template, it's easy to change...

I agree. I'm going to merge this now.

**Comment by radical** [comment]
@timheuer could you please answer/address @RussKie's feedback in a follow up PR?

**Pull request merged**